test/test.nim

```diff
@@ -1,3 +1,3 @@
-import random.mersenne
-import random.xorshift
+import random/mersenne
+import random/xorshift
```

Comment:
Is this change necessary?

Reply:
Yes, importing modules using dots has just been deprecated